README.md

```diff
@@ -1,2 +1,6 @@
 # luminosity-led
 Control brightness and color of LEDs
+
+# Install Recommended/Required VSCode Extensions
+https://code.visualstudio.com/docs/editor/extension-marketplace#_recommended-extensions
+Install all of the VSCode extensions 
```

Comment:
I think adding some instructions could help with how to install the VSCode Extensions.
What I did was:
1.  Ctrl + Shift + P (Open the Command Pallete)
2.  Extensions: Show Recommended Extensions
3.  Click on the cloud icon to "Install Workplace Recommended Extensions"